ApiDocs.Console/Program.cs

```diff
@@ -966,6 +966,14 @@ internal static void WriteValidationError(string indent, ValidationError error)
                 return false;
             }
 
+            IServiceAccount secondaryAccount = null;
+            if (options.SecondaryAccountName != null)
+            {
+                var secondaryAccounts = options.FoundAccounts.Where(
+                    x => options.SecondaryAccountName.Equals(x.Name));
+                secondaryAccount = secondaryAccounts.Any() ? secondaryAccounts.First() : null;
```

Comment:
Assuming `null` is the default value, this would be more concise: `secondaryAccounts.FirstOrDefault()`